### PR TITLE
munge-sdp: keep offer/answer objects around

### PR DESCRIPTION
**Pull request by fippo**
IMO the offer and answer objects should be kept around and just the SDP should be replaced.
More detailed rationale should follow soon :-)

### CODE REVIEW

File: src/content/peerconnection/munge-sdp/js/main.js [hunk 1]
```diff
@@ -183,10 +185,8 @@ function onCreateSessionDescriptionError(error) {
 function setOffer() {
   var sdp = offerSdpTextarea.value;
   sdp = maybeAddLineBreakToEnd(sdp);
-  var offer = new RTCSessionDescription({
```
Comment (KaptenJansson) [inline src/content/peerconnection/munge-sdp/js/main.js]: Why do we not need to use the RTCSessionDescription interface anymore?

File: src/content/peerconnection/munge-sdp/js/main.js [hunk 2]
```diff
@@ -183,10 +185,8 @@ function onCreateSessionDescriptionError(error) {
...
-    type: 'offer',
-    sdp: sdp
-  });
+  sdp = sdp.replace(/\n/g, '\r\n');
```
Comment (KaptenJansson) [inline src/content/peerconnection/munge-sdp/js/main.js]: Is this done due to not using RTCSessionDescription anymore?

Comment (fippo) [inline src/content/peerconnection/munge-sdp/js/main.js]: no, just to recreate the CRLF style which is mangled (hah) by the textarea. Now SDP parser are required to deal with both CRLF and LF but I prefer CRLF.

File: src/content/peerconnection/munge-sdp/js/main.js (hunk 1 above)
Comment (fippo) [inline src/content/peerconnection/munge-sdp/js/main.js]: we keep the original session description (including any extra things it may contain) around.

Comment (KaptenJansson) [inline src/content/peerconnection/munge-sdp/js/main.js]: I see, do you know the historic reason for why we did not?

File: src/content/peerconnection/munge-sdp/js/main.js (hunk 2 above)
Comment (KaptenJansson) [inline src/content/peerconnection/munge-sdp/js/main.js]: I see. Thanks for explaining.

**Comment by KaptenJansson** [comment]
LGTM

**Pull request merged**